### PR TITLE
EASY-1049 upgrade scallop (and check of readme)

### PR DESCRIPTION
**Pull request by jo-pol**
fixes EASY-1049: Upgrade scallop
#### When applied it will
- use the version 1.0.1 of scallop
- improve the stack trace when trying `run.sh --help` is tried without a valid `{{app.home}}/cfg/application.properties`
#### Where should the reviewer @DANS-KNAW/easy start?

Conf and ConfSpec
#### How should this be manually tested?
- execute `./run.sh --help` without providing app.home
- the cause in the stack trace should explain the problem
#### related pull requests on github

| repo | PR |
| --- | --- |
| easy-satge-dataset | [74](https://github.com/DANS-KNAW/easy-stage-dataset/pull/74) |

### CODE REVIEW

**Comment by jo-pol** [comment]
The manual test worked as predicted when compiled with IntelliJ, when compiled with maven I get `ClassNotFoundException`. Anyone an idea why?

File: src/main/scala/nl/knaw/dans/easy/ingest/Conf.scala [hunk 1]
```diff
@@ -29,28 +32,30 @@ class Conf(args: Seq[String], props: PropertiesConfiguration) extends ScallopCon
                 |
                 |Usage:
                 |
-                | $printedName [-u <user> -p <password>] [-f <fcrepo-server>][-i] \\
+                | $printedName [-u <user> -p <password>] [-f <fcrepo-server>] [-i] \\
                 | $indent_____ [<staged-digital-object>... | <staged-digital-object-set>]
                 |
                 |Options:
                 |""".stripMargin)
+  private val s = " provided, please check {{app.home}}/cfg/application.properties"
```
Comment (rvanheest) [inline src/main/scala/nl/knaw/dans/easy/ingest/Conf.scala:40]: name of this value might be confusing to the reader as you have also used `s` in the pattern match on [line 44](https://github.com/DANS-KNAW/easy-ingest/pull/41/files#diff-79bfee1b9ba3798ce350de0651bf6808R44) and [line 51](https://github.com/DANS-KNAW/easy-ingest/pull/41/files#diff-79bfee1b9ba3798ce350de0651bf6808R51). Maybe rename this value to `tailMessage` or something? (_first name I could think of, but you can probably come up with something better..._)

File: src/test/scala/nl/knaw/dans/easy/ingest/ConfSpec.scala [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      new Conf("args".split(" "), props).printHelp()
+    }
+    mockedStdOut.toString
+  }
+
+  /* check for options in help info is spoilt by links in readme
+   * and defaults from {{app.home}}/cfg/application.properties
+   * requiring a semantically valid fedora URL
+   */
+  ignore should "be part of README.md" in {
```
Comment (rvanheest) [inline src/test/scala/nl/knaw/dans/easy/ingest/ConfSpec.scala]: test ignored???

File: src/main/scala/nl/knaw/dans/easy/ingest/Conf.scala (hunk 1 above)
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/ingest/Conf.scala:40]: I don't think this message suffix really adds anything useful. If you must add it, just repeat it in the strings below, and don't make the code harder to read by putting it into a variable 10 lines away.

File: src/test/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (janvanmansum) [inline src/test/resources/logback.xml:1]: Please, turn logging completely off in the test. You can always run the test with a different logback config when you are developing. I would like the build output to remain clean by default. For instance, if a test provokes an error on purpose, I don't want to see any error logging, as the "error" is actually "good" in that context.

**Pull request merged**